next.config.js

```diff
@@ -27,6 +27,12 @@ const nextConfig = {
         port: "",
         pathname: "/a/**",
       },
+      {
```

Comment:
this is the same as the config above, you should be able to combine them

Reply:
no i tried, and they are different, one is a, one is a-